deploy/post_deploy_testing/cypress/e2e/07_higlass_display.cy.js

```diff
@@ -159,8 +172,9 @@ describe("HiGlass Display pages", function(){
             cy.visit(draftUrl);
 
             // There will be an AJAX response to a POST for the new Higlass display, so capture it here.
-            cy.server();
-            cy.route('POST', '/higlass-view-configs/').as('newHiglassDisplay');
+            cy.intercept('POST', '/higlass-view-configs/').as('newHiglassDisplay');
+            // cy.server();
+            // cy.route('POST', '/higlass-view-configs/').as('newHiglassDisplay');
```

Comment:
If this (and other similar comments) aren't necessary anymore, might be good to get rid of it; if it's needed again, it'll always be in git history.